test/stub-test.js

```diff
@@ -1027,11 +996,7 @@ describe("stub", function () {
                     fail("should not call getter");
                 }
             };
-            var stub = createStub(obj, "prop", {get: function () {
-                return 43;
-            }});
-
-            assert.equals(obj.prop, 43);
+            var stub = createStub(obj, "prop");
```

Comment:
That was a mistake! I've fixed this in the offending commit and have rebased my branch